src/ODETools/ODESolvers.jl

```diff
@@ -23,6 +23,30 @@ function solve(
   GenericODESolution(solver,op,u0,t0,tf)
 end
 
+# Default API 2nd Order ODE
+
+function solve_step!(
+  uF::AbstractVector,
+  solver::ODESolver,
+  op::SecondOrderODEOperator,
+  u0::AbstractVector,
+  v0::AbstractVector,
+  a0::AbstractVector,
+  t0::Real) # -> (uF,tF,cache)
+  solve_step!(uF,solver,op,u0,v0,a0,t0,nothing)
+end
+
```

Comment:
Why `a0` is needed here? In general, you only need first order derivatives as initial value of a second order operator. Besides, it is not consistent with the first order case.

If we can eliminate `a0`, again, working with a template of N-vectors, I think we would not need to replicate code, one implementation would be enough.

Reply:
The general Newmark-beta method needs `u0`, `v0` and `a0`. It is also consistent with other multi-step methods for first order ODEs like the generalized-alpha, which needs `u0` and `v0`, or BDF2 ...

The new implementation can handle an arbitrary number of initial values.